control/reporter.go

```diff
@@ -0,0 +1,32 @@
+package control
+
+import (
+	"github.com/opencontrol/fedramp-templater/reporter"
+	"io"
+	"fmt"
+	"strings"
+)
+
+type diffReporter struct {
+	controlName string
+	field string
+	sspValue string
+	yamlValue string
+}
+
+// NewDiff creates a new collection of information that can report diff info for a control.
+func NewDiff(controlName, field, sspValue, yamlValue string) reporter.Reporter {
+	return diffReporter{
+		controlName: strings.TrimSpace(controlName),
+		field: strings.TrimSpace(field),
+		sspValue: strings.TrimSpace(sspValue),
+		yamlValue: strings.TrimSpace(yamlValue),
+	}
+}
+
+// WriteTextTo writes diff information for a control to the writer in plain text format.
+func (r diffReporter) WriteTextTo(writer io.Writer) error {
```

Comment:
Since we are not passing large amounts of information around and we are only reading, we can use a receiver by value